srcpkgs/batsignal/template

```diff
@@ -0,0 +1,18 @@
+# Template file for 'batsignal'
+pkgname=batsignal
+version=1.1.2
+revision=1
+archs=noarch
+build_style=gnu-makefile
+makedepends="libnotify-devel make pkg-config libglib-devel glib-devel"
```

Comment:
* remove make
* move pkg-config and glib-devel to hostmakedepends

Reply:
Wow, I missed that it was in makedepends and not hostmakedepends.